modules/pipeline/providers/dispatcher/hash.go

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) 2021 Terminus, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package dispatcher
+
+import (
+	"github.com/cespare/xxhash/v2"
+)
+
+type defaultHash struct{}
+
+func (h defaultHash) Sum64(bytes []byte) uint64 {
+	return xxhash.Sum64(bytes)
+}
```

Comment:
should encapsulate a common pkg of hashing

Reply:
no need.